scripts/sct_analyze_lesion.py

```diff
@@ -485,47 +504,41 @@ def ifolder2tmp(self):
         os.chdir(self.tmp_dir)  # go to tmp directory
 
 
-def main(args=None):
-    if args is None:
-        args = sys.argv[1:]
-
-    # get parser
-    parser = get_parser()
-    arguments = parser.parse(args)
+def main(arguments):
 
     # set param arguments ad inputted by user
-    fname_mask = arguments["-m"]
+    fname_mask = arguments.m
 
     # SC segmentation
-    if '-s' in arguments:
-        fname_sc = arguments["-s"]
+    if vars(arguments)["s"] != None:
```

Comment:
You can use `arguments.s is not None`. Specifying an argument has it default to None.